galaxy_ng/app/api/v3/views/excludes.py

```diff
@@ -0,0 +1,85 @@
+import yaml
+from django.core.exceptions import ObjectDoesNotExist
+from galaxy_ng.app import models
+from galaxy_ng.app.access_control import access_policy
+from galaxy_ng.app.api import base as api_base
+from pulp_ansible.app.models import AnsibleDistribution
+from rest_framework.renderers import (
+    BaseRenderer,
+    BrowsableAPIRenderer,
+    JSONRenderer
+)
+from rest_framework.request import Request
+from rest_framework.response import Response
+from yaml.dumper import SafeDumper
+
+
+def get_synclist_excludes(base_path):
+    """Get Synclist from distro base_path"""
+    try:
+        repo = AnsibleDistribution.objects.get(base_path=base_path).repository
+        synclist = models.SyncList.objects.get(repository=repo, policy="exclude")
+        return synclist.collections.all()
+    except ObjectDoesNotExist:
+        return None
+
+
+def serialize_collection_queryset(queryset):
+    """Serialize a Queryset in to a JSONable format."""
+    return queryset is not None and [
+        {
+            "name": "{collection.namespace}/{collection.name}".format(collection=collection)
```

Comment:
The requirements.yml format uses `namespace.name`, not `namespace/name`.

Reply:
@newswangerd fixed